artifacts/kernel/kernel/pkg.yaml

```diff
@@ -5,18 +5,34 @@ dependencies:
   - stage: kernel-build
 steps:
   - env:
-      ARCH: arm64
+      ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
     install:
       - |
         cd /src
 
         mkdir -p /rootfs/boot
-        mkdir -p /rootfs/dtb/rockchip
+        mkdir -p /rootfs/dtb
         mkdir -p /rootfs/certs
         cp /src/certs/signing_key.x509 /rootfs/certs/signing_key.x509
-
-        mv arch/arm64/boot/Image /rootfs/boot/vmlinuz
-        mv arch/arm64/boot/dts/rockchip/rk3588-turing-rk1.dtb /rootfs/dtb/rockchip/rk3588-turing-rk1.dtb
+        case $ARCH in
+            x86_64)
+                mv arch/x86/boot/bzImage /rootfs/boot/vmlinuz
+            ;;
+            arm64)
+                mv arch/arm64/boot/vmlinuz.efi /rootfs/boot/vmlinuz
```

Comment:
That's incorrect since the arm64 build doesn't provide such file. Instead it's called Image.